stats/record.go

```diff
@@ -96,7 +96,12 @@ func Record(ctx context.Context, ms ...Measurement) {
 	if len(ms) == 0 {
 		return
 	}
-	recorder := internal.MeasurementRecorder.(measurementRecorder)
+	recorder, initialized := internal.MeasurementRecorder.(measurementRecorder)
+	if !initialized {
+		// the init function in view/worker.go has not bee run yet.  It is
```

Comment:
```suggestion
		// The init function in view/worker.go has not been run yet.  It is
```

typo: `bee` -> `been` 

capitalization: `// the` -> `// The` (to be consistent with project comment style)